onadata/libs/authentication.py

```diff
@@ -114,14 +121,23 @@ def authenticate(self, request):
             remaining_attempts = getattr(settings, "MAX_LOGIN_ATTEMPTS", 10) - attempts
             # pylint: disable=unused-variable
             lockout_time = getattr(settings, "LOCKOUT_TIME", 1800) // 60  # noqa
-            raise AuthenticationFailed(
-                _(
-                    "Invalid username/password. "
-                    f"For security reasons, after {remaining_attempts} more failed "
-                    f"login attempts you'll have to wait {lockout_time} minutes "
-                    "before trying again."
-                )
+            ip_address, username = retrieve_user_identification(request)
+            user_agent = request.META.get("HTTP_USER_AGENT", None)
+            info_str = (
+                f"IP: {ip_address}, USERNAME: {username}, "
+                f"REMAINING_ATTEMPTS: {remaining_attempts}, USER_AGENT: {user_agent}"
             )
+            logger.info(info_str)
```

Comment:
This information would only be relevant when debugging a service and not always. I recommend we use `logger.debug()`.